### DISCUSSION TITLE
2010/08/06/%E9%98%9F%E5%88%97%E6%B6%88%E6%81%AF%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%8A%B6%E6%80%81%E6%9C%BA%E4%B8%8E%E4%BA%8B%E4%BB%B6%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%8A%B6%E6%80%81%E6%9C%BA%E7%9A%84%E5%AF%B9%E6%AF%94

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#293] [Announcements]
# 2010/08/06/%E9%98%9F%E5%88%97%E6%B6%88%E6%81%AF%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%8A%B6%E6%80%81%E6%9C%BA%E4%B8%8E%E4%BA%8B%E4%BB%B6%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%8A%B6%E6%80%81%E6%9C%BA%E7%9A%84%E5%AF%B9%E6%AF%94

一般来说一个程序的主程序，就是他的主界面。编写这样的程序是有固定套路的，也就是它的设计模式。

https://blog.qizhen.xyz/2010/08/06/%E9%98%9F%E5%88%97%E6%B6%88%E6%81%AF%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%8A%B6%E6%80%81%E6%9C%BA%E4%B8%8E%E4%BA%8B%E4%BB%B6%E9%A9%B1%E5%8A%A8%E7%9A%84%E7%8A%B6%E6%80%81%E6%9C%BA%E7%9A%84%E5%AF%B9%E6%AF%94

**Comment by ruanqizhen** [comment]
之前博客下的留言：

ruanqizhen
2011年03月25日 16:46 编辑
以后多交流

回复
agilentvee
2011年03月25日 15:59 编辑
谢谢阮大哥的建议，这个PPT也是我初次尝试写的。小弟只是一个菜鸟，labview方面希望大哥能多多指点

回复
ruanqizhen
2011年03月25日 13:51 编辑
虽然没有时间研究你的程序，但是我仔细看了你写的PPT，你的想法还是比较清楚可行的。不过PPT不适合用于和网友交流，以后有时间写成文章的形式，发到论坛里，会收到更多反馈的。

回复
agilentvee
2011年03月24日 18:01 编辑
就好像等高考分数一样的心情等大哥给予指点啊

回复
agilentvee
2011年03月22日 20:41 编辑
http://www.ourdev.cn/bbs/bbs_content.jsp?bbs_sn=4628643&bbs_page_no=1&bbs_id=9999

回复
ruanqizhen
2011年03月21日 19:20 编辑
有链接吗？

回复
我的一些个人看法
2011年03月21日 15:29 编辑
在这方面小弟也写过一篇文章《基于VI服务器与队列的程序架构课件 》希望得到阮大哥的指正

回复
竹贤
2010年08月6日 20:17 编辑
事件驱动那一列的文字无法读全，被边框删去了部分。

回复
